docs/generated/sql/bnf/stmt_block.bnf

```diff
@@ -1306,6 +1306,7 @@ unreserved_keyword ::=
 	| 'NOWAIT'
 	| 'NULLS'
 	| 'IGNORE_FOREIGN_KEYS'
```

Comment:
notes on commit message: while LDR is in private preview, no need to include a release note.